<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>160</width>
          <height>551</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="navmenu" native="true">
           <property name="styleSheet">
            <string notr="true">#navmenu{
	background-color: #FCF8DD;
	border: 1px solid black; 	
}
</string>
           </property>
           <widget class="QPushButton" name="btnMenu1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>141</width>
              <height>37</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #2980b9;
}</string>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnMenu2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>141</width>
              <height>37</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #2980b9;
}</string>
            </property>
            <property name="text">
             <string>Translator</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>131</width>
              <height>17</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Aplikasi Sekolah</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>561</width>
          <height>451</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FCF8DD;
</string>
        </property>
        <widget class="QWidget" name="pageMenu1">
         <property name="styleSheet">
          <string notr="true">#pageMenu1{
    background-color: #FCF8DD;
    border: 1px solid black;
}</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>54</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Menu 1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>190</y>
            <width>301</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="midLineWidth">
           <number>-2</number>
          </property>
          <property name="text">
           <string>Welcome To Aplikasi 
SDIT NurHikmah</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pageMenu2">
         <property name="styleSheet">
          <string notr="true">#pageMenu2 {
    background-color: #FCF8DD;
    border: 1px solid black;
}</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>54</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Menu 2</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="txtIndo">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>521</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #ffffff
</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="txtEng">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>220</y>
            <width>521</width>
            <height>101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #f0f0f0
</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Bahasa Indonesia</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>200</y>
            <width>91</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Bahasa Inggris</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnTranslate">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>340</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #2980b9;
}</string>
          </property>
          <property name="text">
           <string>Translate</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
